src/main/java/com/conveyal/datatools/manager/jobs/MergeFeedsJob.java

```diff
@@ -280,25 +342,16 @@ public void jobFinished() {
         }
     }
 
-    /**
-     * Collect zipFiles for each feed version before merging tables.
-     * Note: feed versions are sorted by first calendar date so that future dataset is iterated over first. This is
-     * required for the MTC merge strategy which prefers entities from the future dataset over past entities.
-     */
-    private List<FeedToMerge> collectAndSortFeeds(Set<FeedVersion> feedVersions) {
-        return feedVersions.stream().map(version -> {
-            try {
-                return new FeedToMerge(version);
-            } catch (Exception e) {
-                LOG.error("Could not create zip file for version: {}", version.version);
-                return null;
+    private boolean stopTimesMatch(List<StopTime> futureStopTimes, List<StopTime> pastStopTimes) {
+        if (futureStopTimes.size() != pastStopTimes.size()) {
+            return false;
+        }
+        for (int i = 0; i < pastStopTimes.size(); i++) {
+            if (pastStopTimes.get(i).hashCode() != futureStopTimes.get(i).hashCode()) {
```

Comment:
The hashCode isn't necessarily going to ensure that two entities equal each other, just that their hash codes match. It'd probably be better to just do:

```suggestion
            if (pastStopTimes.get(i).equals(futureStopTimes.get(i))) {
```